src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -53,8 +54,12 @@ export default class ContractStore extends BaseStore {
         this.forget_id = contract_info.id;
         const end_time = getEndTime(contract_info);
         const should_update_chart_type = !contract_info.tick_count && !this.is_granularity_set;
+        
+        if (!end_time) this.is_on_going_contract = true;
 
         if (end_time) {
+            SmartChartStore.is_static_chart = !this.is_on_going_contract;
```

Comment:
We usually don't update the store directly, can you add a @action.bound function that sets the is static chart?